contribs/gmf/examples/hidelayertree.inc.css

```diff
@@ -0,0 +1,3 @@
+gmf-layertree {
+  display: none !important;
+}
```

Comment:
In some examples, we have to include the gmf-layertree otherwise we would not have any layers created from the loaded theme.  However, we do not want to showcase the layertree itself, just a specific feature.

For example, in the `editfeatureselector` example.